build/images/Dockerfile.build.agent.coverage

```diff
@@ -24,9 +24,9 @@ RUN go mod download
 
 COPY . /antrea
 
-RUN make antctl-linux antctl-instr-binary && mv bin/antctl-linux bin/antctl
+RUN make antctl-instr-binary && mv bin/antctl-coverage bin/antctl
```

Comment:
I think you can update the Makefile to generate the binary without `-coverage` directory.
@antoninbas  what's your thoughts?

Reply:
I'd prefer to keep it this way to avoid confusion between the 2 binaries. But I don't feel strongly either way.